src/org/team1540/robot2017/CANTalonRamped.java

```diff
@@ -0,0 +1,38 @@
+package org.team1540.robot2017;
+
+import com.ctre.CANTalon;
+
+public class CANTalonRamped extends CANTalon {
+    public CANTalonRamped(int deviceNumber) {
+        super(deviceNumber);
+    }
+
+    private float last = 0;
+
+    public void set(double speed, double limit) {
+        this.set(updateRamping((float) speed, (float) limit));
+    }
+
+    private float updateRamping(float speed, float limit) {
```

Comment:
We don't know how long this will be called, so you might want to check it with System.nanoTime().